examples/vision/aodnet.py

```diff
@@ -0,0 +1,354 @@
+"""
+Title: An All-in-One Network for Dehazing and Beyond
+Author: [Soumik Rakshit](https://github.com/soumik12345)
+Date created: 2021/04/10
+Last modified: 2021/04/10
+Description: Train a lightweight Deep CNN for image denhazing.
+"""
+"""
+# Introduction
+
+Image Dehazing is an important problem is Computer Vision. It can not only be used to
+enhance photographs clicked in hazy conditions, but also can be used to augment other
+computer vision systems such as Object Detection and Seantic Segmentation as a
```

Comment:
Suggestion:

- Use sentence case (insteda, we could add some kind of _emphasis_ on non-commonly capitalized words)
- Fix repetition ("can be used to" x 2)

```suggestion
_Image dehazing_ is an important problem is computer vision that can be used not 
only to enhance photographs clicked in hazy conditions, but also can be used to
augment other computer vision systems, such as _object detection_ and _seantic
segmentation as a
```